src/Actions/Admin/Find.cs

```diff
@@ -0,0 +1,90 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="Find.cs" company="WheelMUD Development Team">
+//   Copyright (c) WheelMUD Development Team.  See LICENSE.txt.  This file is
+//   subject to the Microsoft Public License.  All other rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WheelMUD.Core;
+
+namespace WheelMUD.Actions
+{
+    /// <summary>A command that allows an admin to locate an entity.</summary>
+    [CoreExports.GameAction(0)]
+    [ActionPrimaryAlias("find", CommandCategory.Builder)]
+    [ActionDescription("Finds Things that match the specified argument.")]
+    [ActionSecurity(SecurityRole.minorBuilder)]
+    public class Find : GameAction
+    {
+        /// <summary>List of reusable guards which must be passed before action requests may proceed to execution.</summary>
+        private static readonly List<CommonGuards> ActionGuards = new List<CommonGuards>
+        {
+            CommonGuards.RequiresAtLeastOneArgument
+        };
+
+        private static readonly ThingManager _thingManager = ThingManager.Instance;
+
+        public override void Execute(ActionInput actionInput)
+        {
```

Comment:
A lot of commands start like:
```
var session = actionInput.Session;
if (session == null) return; // This action only makes sense for player sessions.
```
Since this command also only makes sense for player (and not, say, mobile AI or something down the road), this can help simplify your `actionInput.Session?.` into `session.`  :)

Reply:
Added the already built Guard for checking for player, is that enough?

Reply:
Yes, thanks